frontend/src/scenes/batch_exports/BatchExportBackfillModal.tsx

```diff
@@ -53,6 +54,11 @@ export function BatchExportBackfillModal(): JSX.Element {
                 enableFormOnSubmit
                 className="space-y-2"
             >
+                <LemonBanner type="warning">
+                    Creating exports larger than 1 month is not recommended. We are actively working on increasing this
```

Comment:
Since we renamed the modal, let's also rename this: 
```suggestion
                    Exporting historical data beyond 1 month is not recommended. We are actively working on increasing this
```

Reply:
@MarconLP I'll go ahead and merge this one so that I can approve the PR. I hope it's not a big change, but I'll leave merging up to you so that you can revert this if you think it's not appropriate.